cli/Cargo.toml

```diff
@@ -27,6 +27,7 @@ path = "./bench/lsp_bench_standalone.rs"
 
 [build-dependencies]
 deno_broadcast_channel = { version = "0.64.0", path = "../ext/broadcast_channel" }
+deno_cache = { version = "0.1.0", path = "../ext/cache" }
```

Comment:
@dsherret just to double check, do we need to publish manually first? Or are the automation scripts handling this?